config/settings.toml

```diff
@@ -55,3 +55,6 @@ _remove_workspace = true
 # that ansible-playbook accepts for the --extra-vars parameter
 # See https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#defining-variables-at-runtime  # noqa
 ansible_extra_vars = ""
+
+# registry cache IP
```

Comment:
nit: maybe mention containers so it's clear what type of registry we are talking about?

Reply:
Done